packages/medusa/oas/admin-spec3-base.yaml

```diff
@@ -231,65 +231,65 @@ tags:
-  - name: Claim
+  - name: Claims
     description: Claim endpoints that allow handling claims in Medusa.
-  - name: Collection
+  - name: Collections
     description: Collection endpoints that allow handling collections in Medusa.
-  - name: Customer
+  - name: Customers
     description: Customer endpoints that allow handling customers in Medusa.
-  - name: Customer Group
+  - name: Customer Groups
     description: Customer Group endpoints that allow handling customer groups in Medusa.
-  - name: Discount
+  - name: Discounts
     description: Discount endpoints that allow handling discounts in Medusa.
-  - name: Discount Condition
+  - name: Discount Conditions
     description: Discount Condition endpoints that allow handling discount conditions
       in Medusa.
-  - name: Draft Order
+  - name: Draft Orders
     description: Draft Order endpoints that allow handling draft orders in Medusa.
-  - name: Gift Card
+  - name: Gift Cards
     description: Gift Card endpoints that allow handling gift cards in Medusa.
-  - name: Invite
+  - name: Invites
     description: Invite endpoints that allow handling invites in Medusa.
-  - name: Note
+  - name: Notes
     description: Note endpoints that allow handling notes in Medusa.
-  - name: Notification
+  - name: Notifications
     description: Notification endpoints that allow handling notifications in Medusa.
-  - name: Order
+  - name: Orders
     description: Order endpoints that allow handling orders in Medusa.
-  - name: Price List
+  - name: Price Lists
     description: Price List endpoints that allow handling price lists in Medusa.
-  - name: Product
+  - name: Products
     description: Product endpoints that allow handling products in Medusa.
-  - name: Product Tag
+  - name: Product Tags
     description: Product Tag endpoints that allow handling product tags in Medusa.
-  - name: Product Type
+  - name: Product Types
     description: Product Types endpoints that allow handling product types in Medusa.
-  - name: Product Variant
+  - name: Product Variants
     description: Product Variant endpoints that allow handling product variants in Medusa.
-  - name: Region
+  - name: Regions
     description: Region endpoints that allow handling regions in Medusa.
-  - name: Return Reason
+  - name: Return Reasons
     description: Return Reason endpoints that allow handling return reasons in Medusa.
-  - name: Return
+  - name: Returns
     description: Return endpoints that allow handling returns in Medusa.
-  - name: Sales Channel
+  - name: Sales Channels
     description: Sales Channel endpoints that allow handling sales channels in Medusa.
-  - name: Shipping Option
+  - name: Shipping Options
     description: Shipping Option endpoints that allow handling shipping options in Medusa.
-  - name: Shipping Profile
+  - name: Shipping Profiles
     description: Shipping Profile endpoints that allow handling shipping profiles in
       Medusa.
   - name: Store
```

Comment:
Should it be pluralized too?

Reply:
I believe that Store should remain singular since its path url a singular. e.g.: `/admin/store/payment-providers`.
And the JS client is also declaring store as singular.
https://github.com/medusajs/medusa/blob/e670ed2a1abd6415d29d9782c9d0d91fbb5c7588/packages/medusa-js/src/resources/admin/index.ts#L68-L68